### PR TITLE
Bugfix in choosing whether to open new game tab or not

### PR DESCRIPTION
**Pull request by GreenAsJade**
Fixes [the problem of opening a new tab if on a game that is finished](https://forums.online-go.com/t/clicking-on-black-dot-with-of-moves-waiting-opens-in-new-tab/32626/15?u=eugene) which should not happen

## Proposed Changes
 
Look in the right place for window's game status.

### CODE REVIEW

**Comment by GreenAsJade** [comment]
Not sure how that originally got through my testing 🤦‍♀️

File: src/components/Notifications/Notifications.tsx [hunk 1]
```diff
@@ -280,7 +280,7 @@ class NotificationManager {
         const goban = window["global_goban"];
```
Comment (benjaminpjones) [inline src/components/Notifications/Notifications.tsx:280]: I strongly recommend specifying type when accessing variables from `window`:

```
const goban = window["global_goban"] as Goban;
```

Comment (GreenAsJade) [reply src/components/Notifications/Notifications.tsx]: Done

**Comment by anoek** [comment]
Thanks for the fix!

**Pull request merged**